axelrod/tests/strategies/test_defector.py

```diff
@@ -44,9 +44,18 @@ def test_strategy(self):
         # Starts by defecting.
         self.first_play_test(D)
         self.second_play_test(D, D, D, D)
-        # Test if strategy tries to trick opponent.
-        self.responses_test([D], [C, C, C], [C, C, C])
-        self.responses_test([D], [C, C, C, D, D], [C, C, C, C, D])
-        history = [C, C, C, D, D] + [C] * 11
-        opponent_history = [C, C, C, C, D] + [D] + [C] * 10
-        self.responses_test([D], history, opponent_history)
+
+    def test_cooperates_if_opponent_history_has_C_and_last_three_are_D(self):
```

Comment:
is this way ok?  i prefer this to a long list (to me, it more clearly shows the expected behavior), but i'll do long list.